web/src/test/java/org/openmrs/web/filter/util/CustomResourceLoaderTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.web.filter.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.util.LocaleUtility;
+
+import java.util.*;
+
+public class CustomResourceLoaderTest {
+
+	@Test
+	public void testUpdateResource() {
```

Comment:
@ajit-junghare the tests method name should be something like updateResources_shouldUpdateResources()

kindly have a look at this https://wiki.openmrs.org/display/docs/Unit+Testing+With+at-should+Annotation

Reply:
ok .. done